console/algorithms/Cargo.toml

```diff
@@ -14,13 +14,11 @@ harness = false
 [dependencies.snarkvm-console-types]
 path = "../types"
 version = "0.9.13"
-default-features = false
```

Comment:
This can be removed because it's automatically not default because there are features specified below